main_test.go

```diff
@@ -2,12 +2,15 @@ package main
 
 import (
 	"context"
+	"database/sql"
 	"fmt"
+	"gotest.tools/v3/assert"
```

Comment:
We can go with a more idiomatic if-equals instead of a dependency, or use testify